### PR TITLE
#35 add request url to stub

### LINKED ISSUE #35: Add a request url to the stub edit page
For easy coping and paste to a client

### PR DESCRIPTION
**Pull request by PetrS12**
closes #35

### CODE REVIEW

File: http_stubs/templatetags/utils.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True, name='absolute')
+def get_absolute_url_tag(context, url):
+    """Тег возвращающий абсолютный урл."""
```
[the remark this answers is not in the document]
Comment (PetrS12) [reply http_stubs/templatetags/utils.py]: Sorry for my Russian;) It's just a habit.
Corrected.

[the remark this answers is not in the document]
Comment (lowitea) [reply http_stubs/templatetags/utils.py]: Thank you)

File: templates/admin/http_stubs/change_form.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{% extends "admin/change_form.html" %}
+{% load i18n admin_urls static admin_modify simpletags%}
```
Comment (emorozov) [inline templates/admin/http_stubs/change_form.html:2]:
```suggestion
```

Здесь не используется ни один тег (кроме доступных по умолчанию).

File: templates/admin/http_stubs/includes/fieldset.html [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+                        {{ field.field }}{{ field.label_tag }}
+                    {% else %}
+                        {{ field.label_tag }}
+                        {% if field.is_readonly %}
+                            <div class="readonly">{{ field.contents }}</div>
+                        {% else %}
+                            {{ field.field }}
+                            {# added request url for path field(issue #35) #}
+                            {% if field.field.name == 'path' %}<div class="readonly"><div id="full-path">{% absolute field.field.value %}</div><div id="path-copied-alert">Скопировано!</div></div>{% endif %}
+                            {#  #}
```
Comment (emorozov) [inline templates/admin/http_stubs/includes/fieldset.html:24]:
```suggestion
```

**Review by emorozov — APPROVED** [review]

File: templates/admin/http_stubs/change_form.html (hunk 2 above)
Comment (PetrS12) [reply templates/admin/http_stubs/change_form.html]: Thanks! Corrected.

File: templates/admin/http_stubs/includes/fieldset.html (hunk 3 above)
Comment (PetrS12) [reply templates/admin/http_stubs/includes/fieldset.html]: Corrected.

**Review by lowitea — APPROVED** [review]

File: templates/admin/http_stubs/includes/fieldset.html (hunk 3 above)
Comment (emorozov) [reply templates/admin/http_stubs/includes/fieldset.html]: BTW, I've missed Russian word 'Скопировано!' here. :) Probably should be in English or use i18n for translation.

Comment (lowitea) [reply templates/admin/http_stubs/includes/fieldset.html]: Meeh... I has created a new issue for it #50

**Pull request merged**